yb-voyager/cmd/common.go

```diff
@@ -922,6 +923,34 @@ func getImportedSnapshotRowsMap(dbType string) (*utils.StructMap[sqlname.NameTup
 	return snapshotRowsMap, nil
 }
 
+func getImportedSizeMap() (*utils.StructMap[sqlname.NameTuple, int64], error) { //used for import data file case right now
```

Comment:
Isn't there an existing function that we can use for this (that import-data status would be using) ?

Reply:
Not exactly, import-data-status is handling this differently by going to each datafile and preparing table to display. we can't use it.